server/db/schema.js

```diff
@@ -20,6 +20,7 @@ const collections = {
   ROS_STATUS: 'return_of_service_status',
   ADMIN_OPS_AUDIT: 'admin_operation_audit',
   SITE_PARTICIPANTS_STATUS: 'site_participants_status',
+  MOH_EDIT_AUDIT: 'moh_edit_audit',
```

Comment:
Where do we document the purpose of tables? Is it clear enough that this table is only used to track MOH edits on participant applications? If I looked at this table and it's data I feel like I could assume this might track other things MoH might be able to edit (site data for example)

Last comment I think! Sorry for the back and forth